extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -576,45 +605,51 @@ protected function setResponseValuesFromConfig( Wikia\Search\Config $searchConfi
 		if ( $format == 'json' || $format == 'jsonp' ) {
 			$results = $searchConfig->getResults();
 			if ( $results ) {
-				$response->setData( $results->toArray( explode( ',', $this->getVal( 'jsonfields', 'title,url,pageid' ) ) ) );
+				$response->setData(
+					$results->toArray( explode( ',', $this->getVal( 'jsonfields', 'title,url,pageid' ) ) )
+				);
 			} else {
 				$response->setData( [] );
 			}
+
 			return;
 		}
-		if ( ! $searchConfig->getInterWiki() ) {
-			$this->setVal( 'advancedSearchBox', $this->sendSelfRequest( 'advancedBox', array( 'config' => $searchConfig ) ) );
+		if ( !$searchConfig->getInterWiki() ) {
+			$this->setVal(
+				'advancedSearchBox',
+				$this->sendSelfRequest( 'advancedBox', [ 'config' => $searchConfig ] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "config" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)